[2023-03-07T23:59:07.469+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: bitcoin_second_webscrape.google_drive_upload manual__2023-03-07T23:47:42.922440+00:00 [queued]>
[2023-03-07T23:59:07.536+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: bitcoin_second_webscrape.google_drive_upload manual__2023-03-07T23:47:42.922440+00:00 [queued]>
[2023-03-07T23:59:07.540+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-07T23:59:07.543+0000] {taskinstance.py:1280} INFO - Starting attempt 3 of 3
[2023-03-07T23:59:07.551+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-07T23:59:07.774+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonVirtualenvOperator): google_drive_upload> on 2023-03-07 23:47:42.922440+00:00
[2023-03-07T23:59:07.874+0000] {standard_task_runner.py:55} INFO - Started process 907 to run task
[2023-03-07T23:59:07.936+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'bitcoin_second_webscrape', 'google_drive_upload', 'manual__2023-03-07T23:47:42.922440+00:00', '--job-id', '181', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmpzehsa69r']
[2023-03-07T23:59:07.941+0000] {standard_task_runner.py:83} INFO - Job 181: Subtask google_drive_upload
[2023-03-07T23:59:08.513+0000] {task_command.py:388} INFO - Running <TaskInstance: bitcoin_second_webscrape.google_drive_upload manual__2023-03-07T23:47:42.922440+00:00 [running]> on host cd114ba8c1f8
[2023-03-07T23:59:08.969+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=bitcoin_second_webscrape
AIRFLOW_CTX_TASK_ID=google_drive_upload
AIRFLOW_CTX_EXECUTION_DATE=2023-03-07T23:47:42.922440+00:00
AIRFLOW_CTX_TRY_NUMBER=3
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-07T23:47:42.922440+00:00
[2023-03-07T23:59:09.000+0000] {process_utils.py:179} INFO - Executing cmd: /usr/local/bin/python -m virtualenv /tmp/venvkciemkvc --system-site-packages
[2023-03-07T23:59:09.054+0000] {process_utils.py:183} INFO - Output:
[2023-03-07T23:59:13.484+0000] {taskinstance.py:1077} INFO - Dependencies not met for <TaskInstance: bitcoin_second_webscrape.google_drive_upload manual__2023-03-07T23:47:42.922440+00:00 [running]>, dependency 'Task Instance Not Running' FAILED: Task is in the running state
[2023-03-07T23:59:13.494+0000] {taskinstance.py:1077} INFO - Dependencies not met for <TaskInstance: bitcoin_second_webscrape.google_drive_upload manual__2023-03-07T23:47:42.922440+00:00 [running]>, dependency 'Task Instance State' FAILED: Task is in the 'running' state.
[2023-03-07T23:59:13.502+0000] {local_task_job.py:147} INFO - Task is not able to be run
[2023-03-07T23:59:14.006+0000] {process_utils.py:187} INFO - created virtual environment CPython3.7.16.final.0-64 in 2145ms
[2023-03-07T23:59:14.008+0000] {process_utils.py:187} INFO -   creator CPython3Posix(dest=/tmp/venvkciemkvc, clear=False, no_vcs_ignore=False, global=True)
[2023-03-07T23:59:14.031+0000] {process_utils.py:187} INFO -   seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/***/.local/share/virtualenv)
[2023-03-07T23:59:14.041+0000] {process_utils.py:187} INFO -     added seed packages: pip==22.3.1, setuptools==65.6.3, wheel==0.38.4
[2023-03-07T23:59:14.044+0000] {process_utils.py:187} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2023-03-07T23:59:14.259+0000] {process_utils.py:179} INFO - Executing cmd: /tmp/venvkciemkvc/bin/pip install -r /tmp/venvkciemkvc/requirements.txt
[2023-03-07T23:59:14.341+0000] {process_utils.py:183} INFO - Output:
[2023-03-07T23:59:21.407+0000] {process_utils.py:187} INFO - Collecting selenium
[2023-03-07T23:59:21.475+0000] {process_utils.py:187} INFO -   Using cached selenium-4.8.2-py3-none-any.whl (6.9 MB)
[2023-03-07T23:59:21.605+0000] {process_utils.py:187} INFO - Requirement already satisfied: pandas in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venvkciemkvc/requirements.txt (line 2)) (1.3.5)
[2023-03-07T23:59:21.771+0000] {process_utils.py:187} INFO - Collecting gcsfs
[2023-03-07T23:59:21.781+0000] {process_utils.py:187} INFO -   Using cached gcsfs-2023.1.0-py2.py3-none-any.whl (26 kB)
[2023-03-07T23:59:21.787+0000] {process_utils.py:187} INFO - Requirement already satisfied: fsspec in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venvkciemkvc/requirements.txt (line 4)) (2022.11.0)
[2023-03-07T23:59:21.791+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-storage in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venvkciemkvc/requirements.txt (line 5)) (2.7.0)
[2023-03-07T23:59:21.793+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-bigquery in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venvkciemkvc/requirements.txt (line 6)) (2.34.4)
[2023-03-07T23:59:21.797+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-auth in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venvkciemkvc/requirements.txt (line 7)) (2.16.0)
[2023-03-07T23:59:21.800+0000] {process_utils.py:187} INFO - Requirement already satisfied: beautifulsoup4 in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venvkciemkvc/requirements.txt (line 8)) (4.11.1)
[2023-03-07T23:59:21.802+0000] {process_utils.py:187} INFO - Requirement already satisfied: lxml in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venvkciemkvc/requirements.txt (line 9)) (4.9.2)
[2023-03-07T23:59:21.893+0000] {process_utils.py:187} INFO - Collecting html5lib
[2023-03-07T23:59:21.905+0000] {process_utils.py:187} INFO -   Using cached html5lib-1.1-py2.py3-none-any.whl (112 kB)
[2023-03-07T23:59:21.911+0000] {process_utils.py:187} INFO - Requirement already satisfied: pytz in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venvkciemkvc/requirements.txt (line 11)) (2022.7.1)
[2023-03-07T23:59:21.989+0000] {process_utils.py:187} INFO - Collecting tzwhere
[2023-03-07T23:59:21.991+0000] {process_utils.py:187} INFO -   Using cached tzwhere-3.0.3-py3-none-any.whl
[2023-03-07T23:59:22.016+0000] {process_utils.py:187} INFO - Requirement already satisfied: apache-***-providers-google in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venvkciemkvc/requirements.txt (line 13)) (8.8.0)
[2023-03-07T23:59:22.020+0000] {process_utils.py:187} INFO - Requirement already satisfied: apache-***[google] in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venvkciemkvc/requirements.txt (line 14)) (2.5.1)
[2023-03-07T23:59:22.140+0000] {process_utils.py:187} INFO - Collecting openai
[2023-03-07T23:59:22.154+0000] {process_utils.py:187} INFO -   Using cached openai-0.27.0-py3-none-any.whl (70 kB)
[2023-03-07T23:59:22.300+0000] {process_utils.py:187} INFO - Collecting pydrive
[2023-03-07T23:59:22.303+0000] {process_utils.py:187} INFO -   Using cached PyDrive-1.3.1-py3-none-any.whl
[2023-03-07T23:59:22.461+0000] {process_utils.py:187} INFO - Collecting trio~=0.17
[2023-03-07T23:59:22.470+0000] {process_utils.py:187} INFO -   Using cached trio-0.22.0-py3-none-any.whl (384 kB)
[2023-03-07T23:59:22.571+0000] {process_utils.py:187} INFO - Collecting trio-websocket~=0.9
[2023-03-07T23:59:22.581+0000] {process_utils.py:187} INFO -   Using cached trio_websocket-0.9.2-py3-none-any.whl (16 kB)
[2023-03-07T23:59:22.592+0000] {process_utils.py:187} INFO - Requirement already satisfied: urllib3[socks]~=1.26 in /home/***/.local/lib/python3.7/site-packages (from selenium->-r /tmp/venvkciemkvc/requirements.txt (line 1)) (1.26.14)
[2023-03-07T23:59:22.597+0000] {process_utils.py:187} INFO - Requirement already satisfied: certifi>=2021.10.8 in /home/***/.local/lib/python3.7/site-packages (from selenium->-r /tmp/venvkciemkvc/requirements.txt (line 1)) (2022.12.7)
[2023-03-07T23:59:22.654+0000] {process_utils.py:187} INFO - Requirement already satisfied: python-dateutil>=2.7.3 in /home/***/.local/lib/python3.7/site-packages (from pandas->-r /tmp/venvkciemkvc/requirements.txt (line 2)) (2.8.2)
[2023-03-07T23:59:22.661+0000] {process_utils.py:187} INFO - Requirement already satisfied: numpy>=1.17.3 in /home/***/.local/lib/python3.7/site-packages (from pandas->-r /tmp/venvkciemkvc/requirements.txt (line 2)) (1.21.1)
[2023-03-07T23:59:22.696+0000] {process_utils.py:187} INFO - Requirement already satisfied: aiohttp!=4.0.0a0,!=4.0.0a1 in /home/***/.local/lib/python3.7/site-packages (from gcsfs->-r /tmp/venvkciemkvc/requirements.txt (line 3)) (3.8.3)
[2023-03-07T23:59:22.708+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-auth-oauthlib in /home/***/.local/lib/python3.7/site-packages (from gcsfs->-r /tmp/venvkciemkvc/requirements.txt (line 3)) (0.8.0)
[2023-03-07T23:59:22.713+0000] {process_utils.py:187} INFO - Requirement already satisfied: requests in /home/***/.local/lib/python3.7/site-packages (from gcsfs->-r /tmp/venvkciemkvc/requirements.txt (line 3)) (2.28.2)
[2023-03-07T23:59:22.720+0000] {process_utils.py:187} INFO - Requirement already satisfied: decorator>4.1.2 in /home/***/.local/lib/python3.7/site-packages (from gcsfs->-r /tmp/venvkciemkvc/requirements.txt (line 3)) (5.1.1)
[2023-03-07T23:59:22.902+0000] {process_utils.py:187} INFO - Collecting fsspec
[2023-03-07T23:59:22.910+0000] {process_utils.py:187} INFO -   Using cached fsspec-2023.1.0-py3-none-any.whl (143 kB)
[2023-03-07T23:59:23.121+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-core<3.0dev,>=2.3.0 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-storage->-r /tmp/venvkciemkvc/requirements.txt (line 5)) (2.3.2)
[2023-03-07T23:59:23.124+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-resumable-media>=2.3.2 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-storage->-r /tmp/venvkciemkvc/requirements.txt (line 5)) (2.4.0)
[2023-03-07T23:59:23.127+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-storage->-r /tmp/venvkciemkvc/requirements.txt (line 5)) (2.8.2)
[2023-03-07T23:59:23.307+0000] {process_utils.py:187} INFO - Requirement already satisfied: protobuf<4.0.0dev,>=3.12.0 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-bigquery->-r /tmp/venvkciemkvc/requirements.txt (line 6)) (3.20.0)
[2023-03-07T23:59:23.316+0000] {process_utils.py:187} INFO - Requirement already satisfied: grpcio<2.0dev,>=1.38.1 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-bigquery->-r /tmp/venvkciemkvc/requirements.txt (line 6)) (1.51.1)
[2023-03-07T23:59:23.321+0000] {process_utils.py:187} INFO - Requirement already satisfied: packaging<22.0dev,>=14.3 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-bigquery->-r /tmp/venvkciemkvc/requirements.txt (line 6)) (21.3)
[2023-03-07T23:59:23.325+0000] {process_utils.py:187} INFO - Requirement already satisfied: proto-plus<2.0.0dev,>=1.15.0 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-bigquery->-r /tmp/venvkciemkvc/requirements.txt (line 6)) (1.19.6)
[2023-03-07T23:59:23.457+0000] {process_utils.py:187} INFO - Requirement already satisfied: pyasn1-modules>=0.2.1 in /home/***/.local/lib/python3.7/site-packages (from google-auth->-r /tmp/venvkciemkvc/requirements.txt (line 7)) (0.2.8)
[2023-03-07T23:59:23.460+0000] {process_utils.py:187} INFO - Requirement already satisfied: cachetools<6.0,>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from google-auth->-r /tmp/venvkciemkvc/requirements.txt (line 7)) (4.2.2)
[2023-03-07T23:59:23.467+0000] {process_utils.py:187} INFO - Requirement already satisfied: rsa<5,>=3.1.4 in /home/***/.local/lib/python3.7/site-packages (from google-auth->-r /tmp/venvkciemkvc/requirements.txt (line 7)) (4.9)
[2023-03-07T23:59:23.471+0000] {process_utils.py:187} INFO - Requirement already satisfied: six>=1.9.0 in /home/***/.local/lib/python3.7/site-packages (from google-auth->-r /tmp/venvkciemkvc/requirements.txt (line 7)) (1.16.0)
[2023-03-07T23:59:23.494+0000] {process_utils.py:187} INFO - Requirement already satisfied: soupsieve>1.2 in /home/***/.local/lib/python3.7/site-packages (from beautifulsoup4->-r /tmp/venvkciemkvc/requirements.txt (line 8)) (2.3.2.post1)
[2023-03-07T23:59:23.644+0000] {process_utils.py:187} INFO - Collecting webencodings
[2023-03-07T23:59:23.656+0000] {process_utils.py:187} INFO -   Using cached webencodings-0.5.1-py2.py3-none-any.whl (11 kB)
[2023-03-07T23:59:24.564+0000] {process_utils.py:187} INFO - Collecting shapely
[2023-03-07T23:59:24.585+0000] {process_utils.py:187} INFO -   Using cached shapely-2.0.1-cp37-cp37m-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (2.3 MB)
[2023-03-07T23:59:24.902+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-pubsub>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (2.13.5)
[2023-03-07T23:59:24.908+0000] {process_utils.py:187} INFO - Requirement already satisfied: apache-***-providers-common-sql>=1.3.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (1.3.3)
[2023-03-07T23:59:24.913+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-tasks>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (2.10.1)
[2023-03-07T23:59:24.918+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-translate<2.0.0,>=1.5.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (1.7.2)
[2023-03-07T23:59:24.925+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-build>=3.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (3.9.0)
[2023-03-07T23:59:24.928+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-datacatalog>=3.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (3.9.0)
[2023-03-07T23:59:24.932+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-language<2.0.0,>=1.1.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (1.3.2)
[2023-03-07T23:59:24.935+0000] {process_utils.py:187} INFO - Requirement already satisfied: grpcio-gcp>=0.2.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (0.2.2)
[2023-03-07T23:59:24.938+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-dlp<2.0.0,>=0.11.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (1.0.2)
[2023-03-07T23:59:24.940+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-kms>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (2.12.0)
[2023-03-07T23:59:24.943+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-bigtable<2.0.0,>=1.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (1.7.3)
[2023-03-07T23:59:24.949+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-texttospeech<2.0.0,>=0.4.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (1.0.3)
[2023-03-07T23:59:24.952+0000] {process_utils.py:187} INFO - Requirement already satisfied: sqlalchemy-bigquery>=1.2.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (1.5.0)
[2023-03-07T23:59:24.959+0000] {process_utils.py:187} INFO - Requirement already satisfied: gcloud-aio-bigquery>=6.1.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (6.2.0)
[2023-03-07T23:59:24.963+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-spanner<2.0.0,>=1.10.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (1.19.3)
[2023-03-07T23:59:24.969+0000] {process_utils.py:187} INFO - Requirement already satisfied: asgiref>=3.5.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (3.6.0)
[2023-03-07T23:59:24.973+0000] {process_utils.py:187} INFO - Requirement already satisfied: gcloud-aio-auth>=4.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (4.1.5)
[2023-03-07T23:59:24.976+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-orchestration-***<2.0.0,>=1.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (1.4.1)
[2023-03-07T23:59:24.990+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-vision<2.0.0,>=0.35.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (1.0.2)
[2023-03-07T23:59:24.992+0000] {process_utils.py:187} INFO - Requirement already satisfied: gcloud-aio-storage in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (8.0.0)
[2023-03-07T23:59:25.001+0000] {process_utils.py:187} INFO - Requirement already satisfied: httpx in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (0.23.3)
[2023-03-07T23:59:25.008+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-bigquery-datatransfer>=3.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (3.7.0)
[2023-03-07T23:59:25.011+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-workflows<2.0.0,>=0.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (1.7.1)
[2023-03-07T23:59:25.018+0000] {process_utils.py:187} INFO - Requirement already satisfied: looker-sdk>=22.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (22.20.0)
[2023-03-07T23:59:25.020+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-compute<2.0.0,>=0.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (0.7.0)
[2023-03-07T23:59:25.023+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-api-python-client<2.0.0,>=1.6.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (1.12.11)
[2023-03-07T23:59:25.025+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-dataplex>=0.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (1.1.0)
[2023-03-07T23:59:25.029+0000] {process_utils.py:187} INFO - Requirement already satisfied: PyOpenSSL in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (22.1.0)
[2023-03-07T23:59:25.033+0000] {process_utils.py:187} INFO - Requirement already satisfied: json-merge-patch>=0.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (0.2)
[2023-03-07T23:59:25.039+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-memcache>=0.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (1.4.1)
[2023-03-07T23:59:25.042+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-redis>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (2.9.0)
[2023-03-07T23:59:25.047+0000] {process_utils.py:187} INFO - Requirement already satisfied: pandas-gbq in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (0.17.9)
[2023-03-07T23:59:25.051+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-auth-httplib2>=0.0.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (0.1.0)
[2023-03-07T23:59:25.056+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-dataproc>=3.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (5.0.0)
[2023-03-07T23:59:25.059+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-speech<2.0.0,>=0.36.3 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (1.3.4)
[2023-03-07T23:59:25.069+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-dataform>=0.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (0.2.0)
[2023-03-07T23:59:25.072+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-ads>=15.1.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (18.0.0)
[2023-03-07T23:59:25.075+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-container<3.0.0,>=2.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (2.11.1)
[2023-03-07T23:59:25.080+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-dataproc-metastore<2.0.0,>=1.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (1.6.0)
[2023-03-07T23:59:25.087+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-videointelligence<2.0.0,>=1.7.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (1.16.3)
[2023-03-07T23:59:25.091+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-os-login>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (2.7.1)
[2023-03-07T23:59:25.094+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-secret-manager<2.0.0,>=0.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (1.0.2)
[2023-03-07T23:59:25.100+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-monitoring>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (2.11.0)
[2023-03-07T23:59:25.107+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-logging>=2.1.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (3.2.1)
[2023-03-07T23:59:25.113+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-aiplatform<2.0.0,>=1.7.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (1.16.1)
[2023-03-07T23:59:25.118+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-automl>=2.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (2.8.0)
[2023-03-07T23:59:40.901+0000] {process_utils.py:187} INFO - Requirement already satisfied: alembic<2.0,>=1.6.3 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (1.9.2)
[2023-03-07T23:59:40.910+0000] {process_utils.py:187} INFO - Requirement already satisfied: pyjwt>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (2.6.0)
[2023-03-07T23:59:40.928+0000] {process_utils.py:187} INFO - Requirement already satisfied: flask<2.3,>=2.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (2.2.2)
[2023-03-07T23:59:40.933+0000] {process_utils.py:187} INFO - Requirement already satisfied: pathspec~=0.9.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (0.9.0)
[2023-03-07T23:59:40.939+0000] {process_utils.py:187} INFO - Requirement already satisfied: itsdangerous>=2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (2.1.2)
[2023-03-07T23:59:40.943+0000] {process_utils.py:187} INFO - Requirement already satisfied: python-nvd3>=0.15.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (0.15.0)
[2023-03-07T23:59:40.948+0000] {process_utils.py:187} INFO - Requirement already satisfied: tabulate>=0.7.5 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (0.9.0)
[2023-03-07T23:59:40.951+0000] {process_utils.py:187} INFO - Requirement already satisfied: apache-***-providers-http in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (4.1.1)
[2023-03-07T23:59:40.956+0000] {process_utils.py:187} INFO - Requirement already satisfied: psutil>=4.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (5.9.4)
[2023-03-07T23:59:40.959+0000] {process_utils.py:187} INFO - Requirement already satisfied: setproctitle>=1.1.8 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (1.3.2)
[2023-03-07T23:59:40.962+0000] {process_utils.py:187} INFO - Requirement already satisfied: flask-caching>=1.5.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (2.0.2)
[2023-03-07T23:59:40.971+0000] {process_utils.py:187} INFO - Requirement already satisfied: deprecated>=1.2.13 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (1.2.13)
[2023-03-07T23:59:40.974+0000] {process_utils.py:187} INFO - Requirement already satisfied: dill>=0.2.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (0.3.1.1)
[2023-03-07T23:59:40.978+0000] {process_utils.py:187} INFO - Requirement already satisfied: sqlalchemy<2.0,>=1.4 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (1.4.46)
[2023-03-07T23:59:40.984+0000] {process_utils.py:187} INFO - Requirement already satisfied: markdown-it-py>=2.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (2.1.0)
[2023-03-07T23:59:40.993+0000] {process_utils.py:187} INFO - Requirement already satisfied: unicodecsv>=0.14.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (0.14.1)
[2023-03-07T23:59:40.996+0000] {process_utils.py:187} INFO - Requirement already satisfied: apache-***-providers-sqlite in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (3.3.1)
[2023-03-07T23:59:41.003+0000] {process_utils.py:187} INFO - Requirement already satisfied: flask-session>=0.4.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (0.4.0)
[2023-03-07T23:59:41.008+0000] {process_utils.py:187} INFO - Requirement already satisfied: croniter>=0.3.17 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (1.3.8)
[2023-03-07T23:59:41.016+0000] {process_utils.py:187} INFO - Requirement already satisfied: cached-property>=1.5.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (1.5.2)
[2023-03-07T23:59:41.020+0000] {process_utils.py:187} INFO - Requirement already satisfied: lockfile>=0.12.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (0.12.2)
[2023-03-07T23:59:41.026+0000] {process_utils.py:187} INFO - Requirement already satisfied: flask-appbuilder==4.1.4 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (4.1.4)
[2023-03-07T23:59:41.031+0000] {process_utils.py:187} INFO - Requirement already satisfied: configupdater>=3.1.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (3.1.1)
[2023-03-07T23:59:41.038+0000] {process_utils.py:187} INFO - Requirement already satisfied: cryptography>=0.9.3 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (38.0.4)
[2023-03-07T23:59:41.045+0000] {process_utils.py:187} INFO - Requirement already satisfied: linkify-it-py>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (2.0.0)
[2023-03-07T23:59:41.050+0000] {process_utils.py:187} INFO - Requirement already satisfied: attrs>=22.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (22.2.0)
[2023-03-07T23:59:41.063+0000] {process_utils.py:187} INFO - Requirement already satisfied: python-daemon>=2.2.4 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (2.3.2)
[2023-03-07T23:59:41.067+0000] {process_utils.py:187} INFO - Requirement already satisfied: apache-***-providers-ftp in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (3.3.0)
[2023-03-07T23:59:41.075+0000] {process_utils.py:187} INFO - Requirement already satisfied: pygments>=2.0.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (2.14.0)
[2023-03-07T23:59:41.078+0000] {process_utils.py:187} INFO - Requirement already satisfied: jsonschema>=3.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (4.17.3)
[2023-03-07T23:59:41.083+0000] {process_utils.py:187} INFO - Requirement already satisfied: jinja2>=3.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (3.1.2)
[2023-03-07T23:59:41.092+0000] {process_utils.py:187} INFO - Requirement already satisfied: pendulum>=2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (2.1.2)
[2023-03-07T23:59:41.097+0000] {process_utils.py:187} INFO - Requirement already satisfied: termcolor>=1.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (2.2.0)
[2023-03-07T23:59:41.099+0000] {process_utils.py:187} INFO - Requirement already satisfied: graphviz>=0.12 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (0.20.1)
[2023-03-07T23:59:41.127+0000] {process_utils.py:187} INFO - Requirement already satisfied: cron-descriptor>=1.2.24 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (1.2.32)
[2023-03-07T23:59:41.140+0000] {process_utils.py:187} INFO - Requirement already satisfied: gunicorn>=20.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (20.1.0)
[2023-03-07T23:59:41.162+0000] {process_utils.py:187} INFO - Requirement already satisfied: werkzeug>=2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (2.2.2)
[2023-03-07T23:59:41.177+0000] {process_utils.py:187} INFO - Requirement already satisfied: marshmallow-oneofschema>=2.0.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (3.0.1)
[2023-03-07T23:59:41.181+0000] {process_utils.py:187} INFO - Requirement already satisfied: blinker in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (1.5)
[2023-03-07T23:59:41.195+0000] {process_utils.py:187} INFO - Requirement already satisfied: colorlog<5.0,>=4.0.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (4.8.0)
[2023-03-07T23:59:41.199+0000] {process_utils.py:187} INFO - Requirement already satisfied: pluggy>=1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (1.0.0)
[2023-03-07T23:59:41.203+0000] {process_utils.py:187} INFO - Requirement already satisfied: python-slugify>=5.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (7.0.0)
[2023-03-07T23:59:41.208+0000] {process_utils.py:187} INFO - Requirement already satisfied: apache-***-providers-imap in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (3.1.1)
[2023-03-07T23:59:41.210+0000] {process_utils.py:187} INFO - Requirement already satisfied: argcomplete>=1.10 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (2.0.0)
[2023-03-07T23:59:41.213+0000] {process_utils.py:187} INFO - Requirement already satisfied: importlib-metadata>=1.7 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (4.13.0)
[2023-03-07T23:59:41.217+0000] {process_utils.py:187} INFO - Requirement already satisfied: mdit-py-plugins>=0.3.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (0.3.3)
[2023-03-07T23:59:41.221+0000] {process_utils.py:187} INFO - Requirement already satisfied: rich>=12.4.4 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (13.1.0)
[2023-03-07T23:59:41.225+0000] {process_utils.py:187} INFO - Requirement already satisfied: flask-wtf>=0.15 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (1.1.1)
[2023-03-07T23:59:41.230+0000] {process_utils.py:187} INFO - Requirement already satisfied: markupsafe>=1.1.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (2.1.2)
[2023-03-07T23:59:41.239+0000] {process_utils.py:187} INFO - Requirement already satisfied: tenacity>=6.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (8.1.0)
[2023-03-07T23:59:41.245+0000] {process_utils.py:187} INFO - Requirement already satisfied: cattrs>=22.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (22.2.0)
[2023-03-07T23:59:41.251+0000] {process_utils.py:187} INFO - Requirement already satisfied: connexion[flask]>=2.10.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (2.14.1)
[2023-03-07T23:59:41.257+0000] {process_utils.py:187} INFO - Requirement already satisfied: markdown>=3.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (3.4.1)
[2023-03-07T23:59:41.260+0000] {process_utils.py:187} INFO - Requirement already satisfied: flask-login>=0.6.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (0.6.2)
[2023-03-07T23:59:41.269+0000] {process_utils.py:187} INFO - Requirement already satisfied: importlib-resources>=5.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (5.10.2)
[2023-03-07T23:59:41.278+0000] {process_utils.py:187} INFO - Requirement already satisfied: typing-extensions>=4.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (4.4.0)
[2023-03-07T23:59:41.282+0000] {process_utils.py:187} INFO - Requirement already satisfied: lazy-object-proxy in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (1.9.0)
[2023-03-07T23:59:41.288+0000] {process_utils.py:187} INFO - Requirement already satisfied: sqlalchemy-jsonfield>=1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (1.0.1.post0)
[2023-03-07T23:59:41.410+0000] {process_utils.py:187} INFO - Requirement already satisfied: marshmallow<4,>=3 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (3.19.0)
[2023-03-07T23:59:41.414+0000] {process_utils.py:187} INFO - Requirement already satisfied: colorama<1,>=0.3.9 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (0.4.6)
[2023-03-07T23:59:41.421+0000] {process_utils.py:187} INFO - Requirement already satisfied: Flask-Babel<3,>=1 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (2.0.0)
[2023-03-07T23:59:41.428+0000] {process_utils.py:187} INFO - Requirement already satisfied: prison<1.0.0,>=0.2.1 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (0.2.1)
[2023-03-07T23:59:41.431+0000] {process_utils.py:187} INFO - Requirement already satisfied: click<9,>=8 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (8.1.3)
[2023-03-07T23:59:41.435+0000] {process_utils.py:187} INFO - Requirement already satisfied: email-validator<2,>=1.0.5 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (1.3.0)
[2023-03-07T23:59:41.441+0000] {process_utils.py:187} INFO - Requirement already satisfied: Flask-JWT-Extended<5.0.0,>=4.0.0 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (4.4.4)
[2023-03-07T23:59:41.450+0000] {process_utils.py:187} INFO - Requirement already satisfied: WTForms<4 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (3.0.1)
[2023-03-07T23:59:41.458+0000] {process_utils.py:187} INFO - Requirement already satisfied: apispec[yaml]<4,>=3.3 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (3.3.2)
[2023-03-07T23:59:41.467+0000] {process_utils.py:187} INFO - Requirement already satisfied: sqlalchemy-utils<1,>=0.32.21 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (0.39.0)
[2023-03-07T23:59:41.472+0000] {process_utils.py:187} INFO - Requirement already satisfied: marshmallow-enum<2,>=1.5.1 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (1.5.1)
[2023-03-07T23:59:41.478+0000] {process_utils.py:187} INFO - Requirement already satisfied: marshmallow-sqlalchemy<0.27.0,>=0.22.0 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (0.26.1)
[2023-03-07T23:59:41.482+0000] {process_utils.py:187} INFO - Requirement already satisfied: Flask-SQLAlchemy<3,>=2.4 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (2.5.1)
[2023-03-07T23:59:41.977+0000] {process_utils.py:187} INFO - Collecting tqdm
[2023-03-07T23:59:41.993+0000] {process_utils.py:187} INFO -   Using cached tqdm-4.65.0-py3-none-any.whl (77 kB)
[2023-03-07T23:59:42.028+0000] {process_utils.py:187} INFO - Requirement already satisfied: PyYAML>=3.0 in /home/***/.local/lib/python3.7/site-packages (from pydrive->-r /tmp/venvkciemkvc/requirements.txt (line 16)) (6.0)
[2023-03-07T23:59:42.131+0000] {process_utils.py:187} INFO - Collecting oauth2client>=4.0.0
[2023-03-07T23:59:42.139+0000] {process_utils.py:187} INFO -   Using cached oauth2client-4.1.3-py2.py3-none-any.whl (98 kB)
[2023-03-07T23:59:42.223+0000] {process_utils.py:187} INFO - Requirement already satisfied: yarl<2.0,>=1.0 in /home/***/.local/lib/python3.7/site-packages (from aiohttp!=4.0.0a0,!=4.0.0a1->gcsfs->-r /tmp/venvkciemkvc/requirements.txt (line 3)) (1.8.2)
[2023-03-07T23:59:42.226+0000] {process_utils.py:187} INFO - Requirement already satisfied: async-timeout<5.0,>=4.0.0a3 in /home/***/.local/lib/python3.7/site-packages (from aiohttp!=4.0.0a0,!=4.0.0a1->gcsfs->-r /tmp/venvkciemkvc/requirements.txt (line 3)) (4.0.2)
[2023-03-07T23:59:42.231+0000] {process_utils.py:187} INFO - Requirement already satisfied: aiosignal>=1.1.2 in /home/***/.local/lib/python3.7/site-packages (from aiohttp!=4.0.0a0,!=4.0.0a1->gcsfs->-r /tmp/venvkciemkvc/requirements.txt (line 3)) (1.3.1)
[2023-03-07T23:59:42.237+0000] {process_utils.py:187} INFO - Requirement already satisfied: charset-normalizer<3.0,>=2.0 in /home/***/.local/lib/python3.7/site-packages (from aiohttp!=4.0.0a0,!=4.0.0a1->gcsfs->-r /tmp/venvkciemkvc/requirements.txt (line 3)) (2.1.1)
[2023-03-07T23:59:42.241+0000] {process_utils.py:187} INFO - Requirement already satisfied: frozenlist>=1.1.1 in /home/***/.local/lib/python3.7/site-packages (from aiohttp!=4.0.0a0,!=4.0.0a1->gcsfs->-r /tmp/venvkciemkvc/requirements.txt (line 3)) (1.3.3)
[2023-03-07T23:59:42.245+0000] {process_utils.py:187} INFO - Requirement already satisfied: multidict<7.0,>=4.5 in /home/***/.local/lib/python3.7/site-packages (from aiohttp!=4.0.0a0,!=4.0.0a1->gcsfs->-r /tmp/venvkciemkvc/requirements.txt (line 3)) (6.0.4)
[2023-03-07T23:59:42.250+0000] {process_utils.py:187} INFO - Requirement already satisfied: asynctest==0.13.0 in /home/***/.local/lib/python3.7/site-packages (from aiohttp!=4.0.0a0,!=4.0.0a1->gcsfs->-r /tmp/venvkciemkvc/requirements.txt (line 3)) (0.13.0)
[2023-03-07T23:59:42.301+0000] {process_utils.py:187} INFO - Requirement already satisfied: Mako in /home/***/.local/lib/python3.7/site-packages (from alembic<2.0,>=1.6.3->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (1.2.4)
[2023-03-07T23:59:42.379+0000] {process_utils.py:187} INFO - Requirement already satisfied: sqlparse>=0.4.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-common-sql>=1.3.1->apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (0.4.3)
[2023-03-07T23:59:42.738+0000] {process_utils.py:187} INFO - Requirement already satisfied: exceptiongroup in /home/***/.local/lib/python3.7/site-packages (from cattrs>=22.1.0->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (1.1.0)
[2023-03-07T23:59:43.129+0000] {process_utils.py:187} INFO - Requirement already satisfied: clickclick<21,>=1.2 in /home/***/.local/lib/python3.7/site-packages (from connexion[flask]>=2.10.0->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (20.10.2)
[2023-03-07T23:59:43.133+0000] {process_utils.py:187} INFO - Requirement already satisfied: inflection<0.6,>=0.3.1 in /home/***/.local/lib/python3.7/site-packages (from connexion[flask]>=2.10.0->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (0.5.1)
[2023-03-07T23:59:43.297+0000] {process_utils.py:187} INFO - Requirement already satisfied: cffi>=1.12 in /home/***/.local/lib/python3.7/site-packages (from cryptography>=0.9.3->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (1.15.1)
[2023-03-07T23:59:43.437+0000] {process_utils.py:187} INFO - Requirement already satisfied: wrapt<2,>=1.10 in /home/***/.local/lib/python3.7/site-packages (from deprecated>=1.2.13->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (1.14.1)
[2023-03-07T23:59:43.597+0000] {process_utils.py:187} INFO - Requirement already satisfied: cachelib<0.10.0,>=0.9.0 in /home/***/.local/lib/python3.7/site-packages (from flask-caching>=1.5.0->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (0.9.0)
[2023-03-07T23:59:43.836+0000] {process_utils.py:187} INFO - Requirement already satisfied: chardet<4.1,>=2.0 in /home/***/.local/lib/python3.7/site-packages (from gcloud-aio-auth>=4.0.0->apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (4.0.0)
[2023-03-07T23:59:43.844+0000] {process_utils.py:187} INFO - Requirement already satisfied: future<0.19.0,>=0.17.0 in /home/***/.local/lib/python3.7/site-packages (from gcloud-aio-auth>=4.0.0->apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (0.18.3)
[2023-03-07T23:59:43.849+0000] {process_utils.py:187} INFO - Requirement already satisfied: backoff<3.0.0,>=1.0.0 in /home/***/.local/lib/python3.7/site-packages (from gcloud-aio-auth>=4.0.0->apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (1.10.0)
[2023-03-07T23:59:44.051+0000] {process_utils.py:187} INFO - Requirement already satisfied: grpcio-status<2.0.0,>=1.38.1 in /home/***/.local/lib/python3.7/site-packages (from google-ads>=15.1.1->apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (1.48.2)
[2023-03-07T23:59:44.075+0000] {process_utils.py:187} INFO - Requirement already satisfied: googleapis-common-protos<2.0.0,>=1.5.8 in /home/***/.local/lib/python3.7/site-packages (from google-ads>=15.1.1->apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (1.56.4)
[2023-03-07T23:59:44.078+0000] {process_utils.py:187} INFO - Requirement already satisfied: setuptools>=40.3.0 in /tmp/venvkciemkvc/lib/python3.7/site-packages (from google-ads>=15.1.1->apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (65.6.3)
[2023-03-07T23:59:44.930+0000] {process_utils.py:187} INFO - Requirement already satisfied: uritemplate<4dev,>=3.0.0 in /home/***/.local/lib/python3.7/site-packages (from google-api-python-client<2.0.0,>=1.6.0->apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (3.0.1)
[2023-03-07T23:59:44.943+0000] {process_utils.py:187} INFO - Requirement already satisfied: httplib2<1dev,>=0.15.0 in /home/***/.local/lib/python3.7/site-packages (from google-api-python-client<2.0.0,>=1.6.0->apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (0.20.4)
[2023-03-07T23:59:45.131+0000] {process_utils.py:187} INFO - Requirement already satisfied: requests-oauthlib>=0.7.0 in /home/***/.local/lib/python3.7/site-packages (from google-auth-oauthlib->gcsfs->-r /tmp/venvkciemkvc/requirements.txt (line 3)) (1.3.1)
[2023-03-07T23:59:46.135+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-resource-manager<3.0.0dev,>=1.3.3 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-aiplatform<2.0.0,>=1.7.1->apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (1.6.0)
[2023-03-07T23:59:46.584+0000] {process_utils.py:187} INFO - Requirement already satisfied: grpc-google-iam-v1<0.13dev,>=0.12.3 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-bigtable<2.0.0,>=1.0.0->apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (0.12.4)
[2023-03-07T23:59:47.953+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-audit-log<1.0.0dev,>=0.1.0 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-logging>=2.1.1->apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (0.2.4)
[2023-03-07T23:59:47.964+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-appengine-logging<2.0.0dev,>=0.1.0 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-logging>=2.1.1->apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (1.1.3)
[2023-03-07T23:59:51.785+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-crc32c<2.0dev,>=1.0 in /home/***/.local/lib/python3.7/site-packages (from google-resumable-media>=2.3.2->google-cloud-storage->-r /tmp/venvkciemkvc/requirements.txt (line 5)) (1.5.0)
[2023-03-07T23:59:52.847+0000] {process_utils.py:187} INFO - Requirement already satisfied: zipp>=0.5 in /home/***/.local/lib/python3.7/site-packages (from importlib-metadata>=1.7->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (3.11.0)
[2023-03-07T23:59:53.766+0000] {process_utils.py:187} INFO - Requirement already satisfied: pyrsistent!=0.17.0,!=0.17.1,!=0.17.2,>=0.14.0 in /home/***/.local/lib/python3.7/site-packages (from jsonschema>=3.2.0->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (0.19.3)
[2023-03-07T23:59:53.794+0000] {process_utils.py:187} INFO - Requirement already satisfied: pkgutil-resolve-name>=1.3.10 in /home/***/.local/lib/python3.7/site-packages (from jsonschema>=3.2.0->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (1.3.10)
[2023-03-07T23:59:53.950+0000] {process_utils.py:187} INFO - Requirement already satisfied: uc-micro-py in /home/***/.local/lib/python3.7/site-packages (from linkify-it-py>=2.0.0->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (1.0.1)
[2023-03-07T23:59:55.753+0000] {process_utils.py:187} INFO - Requirement already satisfied: mdurl~=0.1 in /home/***/.local/lib/python3.7/site-packages (from markdown-it-py>=2.1.0->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (0.1.2)
[2023-03-07T23:59:56.947+0000] {process_utils.py:187} INFO - Requirement already satisfied: pyasn1>=0.1.7 in /home/***/.local/lib/python3.7/site-packages (from oauth2client>=4.0.0->pydrive->-r /tmp/venvkciemkvc/requirements.txt (line 16)) (0.4.8)
[2023-03-07T23:59:57.064+0000] {process_utils.py:187} INFO - Requirement already satisfied: pyparsing!=3.0.5,>=2.0.2 in /home/***/.local/lib/python3.7/site-packages (from packaging<22.0dev,>=14.3->google-cloud-bigquery->-r /tmp/venvkciemkvc/requirements.txt (line 6)) (3.0.9)
[2023-03-07T23:59:57.360+0000] {process_utils.py:187} INFO - Requirement already satisfied: pytzdata>=2020.1 in /home/***/.local/lib/python3.7/site-packages (from pendulum>=2.0->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (2020.1)
[2023-03-07T23:59:58.910+0000] {process_utils.py:187} INFO - Requirement already satisfied: docutils in /home/***/.local/lib/python3.7/site-packages (from python-daemon>=2.2.4->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (0.19)
[2023-03-07T23:59:59.532+0000] {process_utils.py:187} INFO - Requirement already satisfied: text-unidecode>=1.3 in /home/***/.local/lib/python3.7/site-packages (from python-slugify>=5.0->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (1.3)
[2023-03-07T23:59:59.792+0000] {process_utils.py:187} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.7/site-packages (from requests->gcsfs->-r /tmp/venvkciemkvc/requirements.txt (line 3)) (3.4)
[2023-03-07T23:59:59.942+0000] {process_utils.py:187} INFO - Requirement already satisfied: commonmark<0.10.0,>=0.9.0 in /home/***/.local/lib/python3.7/site-packages (from rich>=12.4.4->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (0.9.1)
[2023-03-08T00:00:01.131+0000] {process_utils.py:187} INFO - Requirement already satisfied: greenlet!=0.4.17 in /home/***/.local/lib/python3.7/site-packages (from sqlalchemy<2.0,>=1.4->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (2.0.1)
[2023-03-08T00:00:01.390+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-bigquery-storage<3.0.0dev,>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from sqlalchemy-bigquery>=1.2.1->apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (2.14.1)
[2023-03-08T00:00:01.395+0000] {process_utils.py:187} INFO - Requirement already satisfied: pyarrow>=3.0.0 in /home/***/.local/lib/python3.7/site-packages (from sqlalchemy-bigquery>=1.2.1->apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (9.0.0)
[2023-03-08T00:00:02.121+0000] {process_utils.py:187} INFO - Requirement already satisfied: sniffio in /home/***/.local/lib/python3.7/site-packages (from trio~=0.17->selenium->-r /tmp/venvkciemkvc/requirements.txt (line 1)) (1.3.0)
[2023-03-08T00:00:02.267+0000] {process_utils.py:187} INFO - Collecting async-generator>=1.9
[2023-03-08T00:00:02.286+0000] {process_utils.py:187} INFO -   Using cached async_generator-1.10-py3-none-any.whl (18 kB)
[2023-03-08T00:00:02.386+0000] {process_utils.py:187} INFO - Collecting outcome
[2023-03-08T00:00:02.394+0000] {process_utils.py:187} INFO -   Using cached outcome-1.2.0-py2.py3-none-any.whl (9.7 kB)
[2023-03-08T00:00:02.402+0000] {process_utils.py:187} INFO - Requirement already satisfied: sortedcontainers in /home/***/.local/lib/python3.7/site-packages (from trio~=0.17->selenium->-r /tmp/venvkciemkvc/requirements.txt (line 1)) (2.4.0)
[2023-03-08T00:00:02.660+0000] {process_utils.py:187} INFO - Collecting wsproto>=0.14
[2023-03-08T00:00:02.674+0000] {process_utils.py:187} INFO -   Using cached wsproto-1.2.0-py3-none-any.whl (24 kB)
[2023-03-08T00:00:03.324+0000] {process_utils.py:187} INFO - Collecting PySocks!=1.5.7,<2.0,>=1.5.6
[2023-03-08T00:00:03.340+0000] {process_utils.py:187} INFO -   Using cached PySocks-1.7.1-py3-none-any.whl (16 kB)
[2023-03-08T00:00:03.757+0000] {process_utils.py:187} INFO - Requirement already satisfied: requests-toolbelt in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-http->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (0.10.1)
[2023-03-08T00:00:04.462+0000] {process_utils.py:187} INFO - Requirement already satisfied: aiofiles<23.0.0,>=0.6.0 in /home/***/.local/lib/python3.7/site-packages (from gcloud-aio-storage->apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (22.1.0)
[2023-03-08T00:00:04.647+0000] {process_utils.py:187} INFO - Requirement already satisfied: rfc3986[idna2008]<2,>=1.3 in /home/***/.local/lib/python3.7/site-packages (from httpx->apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (1.5.0)
[2023-03-08T00:00:04.654+0000] {process_utils.py:187} INFO - Requirement already satisfied: httpcore<0.17.0,>=0.15.0 in /home/***/.local/lib/python3.7/site-packages (from httpx->apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (0.16.3)
[2023-03-08T00:00:05.019+0000] {process_utils.py:187} INFO - Requirement already satisfied: pydata-google-auth in /home/***/.local/lib/python3.7/site-packages (from pandas-gbq->apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (1.5.0)
[2023-03-08T00:00:05.037+0000] {process_utils.py:187} INFO - Requirement already satisfied: db-dtypes<2.0.0,>=0.3.1 in /home/***/.local/lib/python3.7/site-packages (from pandas-gbq->apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (1.0.5)
[2023-03-08T00:00:07.702+0000] {process_utils.py:187} INFO - Requirement already satisfied: pycparser in /home/***/.local/lib/python3.7/site-packages (from cffi>=1.12->cryptography>=0.9.3->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (2.21)
[2023-03-08T00:00:09.742+0000] {process_utils.py:187} INFO - Requirement already satisfied: dnspython>=1.15.0 in /home/***/.local/lib/python3.7/site-packages (from email-validator<2,>=1.0.5->flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (2.3.0)
[2023-03-08T00:00:10.279+0000] {process_utils.py:187} INFO - Requirement already satisfied: Babel>=2.3 in /home/***/.local/lib/python3.7/site-packages (from Flask-Babel<3,>=1->flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvkciemkvc/requirements.txt (line 14)) (2.11.0)
[2023-03-08T00:00:11.214+0000] {process_utils.py:187} INFO - WARNING: google-api-core 2.8.2 does not provide the extra 'grpcgcp'
[2023-03-08T00:00:13.400+0000] {process_utils.py:187} INFO - Requirement already satisfied: anyio<5.0,>=3.0 in /home/***/.local/lib/python3.7/site-packages (from httpcore<0.17.0,>=0.15.0->httpx->apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (3.6.2)
[2023-03-08T00:00:13.407+0000] {process_utils.py:187} INFO - Requirement already satisfied: h11<0.15,>=0.13 in /home/***/.local/lib/python3.7/site-packages (from httpcore<0.17.0,>=0.15.0->httpx->apache-***-providers-google->-r /tmp/venvkciemkvc/requirements.txt (line 13)) (0.14.0)
[2023-03-08T00:00:16.969+0000] {process_utils.py:187} INFO - Requirement already satisfied: oauthlib>=3.0.0 in /home/***/.local/lib/python3.7/site-packages (from requests-oauthlib>=0.7.0->google-auth-oauthlib->gcsfs->-r /tmp/venvkciemkvc/requirements.txt (line 3)) (3.2.2)
[2023-03-08T00:00:24.641+0000] {process_utils.py:187} INFO - Installing collected packages: webencodings, tqdm, shapely, PySocks, outcome, html5lib, fsspec, async-generator, wsproto, tzwhere, trio, oauth2client, trio-websocket, openai, selenium, pydrive, gcsfs
[2023-03-08T00:00:25.737+0000] {process_utils.py:187} INFO -   Attempting uninstall: fsspec
[2023-03-08T00:00:25.742+0000] {process_utils.py:187} INFO -     Found existing installation: fsspec 2022.11.0
[2023-03-08T00:00:25.744+0000] {process_utils.py:187} INFO -     Not uninstalling fsspec at /home/***/.local/lib/python3.7/site-packages, outside environment /tmp/venvkciemkvc
[2023-03-08T00:00:25.747+0000] {process_utils.py:187} INFO -     Can't uninstall 'fsspec'. No files were found to uninstall.
[2023-03-08T00:00:32.480+0000] {process_utils.py:187} INFO - Successfully installed PySocks-1.7.1 async-generator-1.10 fsspec-2023.1.0 gcsfs-2023.1.0 html5lib-1.1 oauth2client-4.1.3 openai-0.27.0 outcome-1.2.0 pydrive-1.3.1 selenium-4.8.2 shapely-2.0.1 tqdm-4.65.0 trio-0.22.0 trio-websocket-0.9.2 tzwhere-3.0.3 webencodings-0.5.1 wsproto-1.2.0
[2023-03-08T00:00:33.158+0000] {process_utils.py:187} INFO - 
[2023-03-08T00:00:33.160+0000] {process_utils.py:187} INFO - [notice] A new release of pip available: 22.3.1 -> 23.0.1
[2023-03-08T00:00:33.166+0000] {process_utils.py:187} INFO - [notice] To update, run: python -m pip install --upgrade pip
[2023-03-08T00:00:33.796+0000] {process_utils.py:179} INFO - Executing cmd: /tmp/venvkciemkvc/bin/python /tmp/venvkciemkvc/script.py /tmp/venvkciemkvc/script.in /tmp/venvkciemkvc/script.out /tmp/venvkciemkvc/string_args.txt
[2023-03-08T00:00:33.848+0000] {process_utils.py:183} INFO - Output:
[2023-03-08T00:00:38.103+0000] {process_utils.py:187} INFO - Traceback (most recent call last):
[2023-03-08T00:00:38.106+0000] {process_utils.py:187} INFO -   File "/tmp/venvkciemkvc/script.py", line 39, in <module>
[2023-03-08T00:00:38.111+0000] {process_utils.py:187} INFO -     res = google_drive_upload(*arg_dict["args"], **arg_dict["kwargs"])
[2023-03-08T00:00:38.116+0000] {process_utils.py:187} INFO -   File "/tmp/venvkciemkvc/script.py", line 36, in google_drive_upload
[2023-03-08T00:00:38.119+0000] {process_utils.py:187} INFO -     file.SetContentFile(os.path.join(data_dir,config['bitcoin_consolidated']))
[2023-03-08T00:00:38.126+0000] {process_utils.py:187} INFO - NameError: name 'config' is not defined
[2023-03-08T00:00:38.763+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 356, in execute
    return super().execute(context=serializable_context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 553, in execute_callable
    return self._execute_python_callable_in_subprocess(python_path, tmp_path)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 418, in _execute_python_callable_in_subprocess
    os.fspath(string_args_path),
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/process_utils.py", line 168, in execute_in_subprocess
    execute_in_subprocess_with_kwargs(cmd, cwd=cwd)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/process_utils.py", line 191, in execute_in_subprocess_with_kwargs
    raise subprocess.CalledProcessError(exit_code, cmd)
subprocess.CalledProcessError: Command '['/tmp/venvkciemkvc/bin/python', '/tmp/venvkciemkvc/script.py', '/tmp/venvkciemkvc/script.in', '/tmp/venvkciemkvc/script.out', '/tmp/venvkciemkvc/string_args.txt']' returned non-zero exit status 1.
[2023-03-08T00:00:38.809+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=bitcoin_second_webscrape, task_id=google_drive_upload, execution_date=20230307T234742, start_date=20230307T235907, end_date=20230308T000038
[2023-03-08T00:00:38.921+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 181 for task google_drive_upload (Command '['/tmp/venvkciemkvc/bin/python', '/tmp/venvkciemkvc/script.py', '/tmp/venvkciemkvc/script.in', '/tmp/venvkciemkvc/script.out', '/tmp/venvkciemkvc/string_args.txt']' returned non-zero exit status 1.; 907)
[2023-03-08T00:00:38.998+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2023-03-08T00:00:39.172+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
